### DISCUSSION TITLE
--relaxed but for everything

### DISCUSSION THREAD

**Discussion by God-damnit-all** [astral-sh/rye#315] [Ideas]
Right now I'm running rye with RYE_TOOLCHAIN set to `C:\Python311\python.exe` and it's working wonderfully. However, I know that at some point there's going to be a new minor upgrade, like to 3.11.5, and I expect that to throw a curveball.

I suggest that if a user wants everything related to a certain major Python version to be redirected to a local install, emulating behavior that's similar to the --relaxed flag, the user could, for example, set RYE_PYTHON311 to `C:\Python311\python.exe` and this would use a cpython@3.11 symlink.

Perhaps there could also be an option to set the RYE_TOOLCHAIN value to `RYE_PYTHON311` to have it use the path in that variable (and of course RYE_PYTHON39, RYE_PYTHON310 etc. would be options as well), and this would cause the installation to create a cpython@3.11 symlink and set self\pyenv.cfg's "version = " line to `3.11`.

**Comment by bluss** [comment]
Why is the minor update a curveball? What's the workflow for a project when you use this relaxed setting, what happens on rye sync?
